app/models/manageiq/providers/ansible_tower/shared/inventory/parser/automation_manager.rb

```diff
@@ -32,6 +32,12 @@ def configuration_scripts
       inventory_object.survey_spec = job_template.survey_spec_hash
       inventory_object.variables = job_template.extra_vars_hash
       inventory_object.inventory_root_group = persister.inventory_root_groups.lazy_find(job_template.inventory_id.to_s)
+      # inventory_object.configuration_script_source = persister.projects.lazy_find(job_template.project_id.to_s)
+      # project = persister.configuration_script_sources.lazy_find(job_template.project_id.to_s)
```

Comment:
@jameswnl cant we store this info too?

`job_template.project` / `configuration_script. configuration_script_source`